CONTRIBUTING.md

```diff
@@ -76,7 +76,7 @@ Before sending us a pull request, please ensure that:
 1. Install tox using `pip install tox`
 1. Install coverage using `pip install .[test]`
 1. cd into the sagemaker-python-sdk folder: `cd sagemaker-python-sdk` or `cd /environment/sagemaker-python-sdk`
-1. Run the following tox command and verify that all code checks and unit tests pass: `tox -- tests/unit`
```

Comment:
Is this correct?
Locally I do `tox -e py310 -- tests/unit`

Reply:
This is a revert so we should reland this update later. I think the -e py310 is optional, just points which py version to use